utils/units.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+import math
+import re
+
+
+FACTOR = 1024
+PREFIXES = ['', 'k', 'M', 'G', 'T']
```

Comment:
should we allow for ```('k', 'K')```? I guess we are ignoring that 'm' and 'M' would be different